front-end/src/main/java/com/okta/spring/example/FrontEndExampleApplication.java

```diff
@@ -44,5 +45,9 @@ String getConfig() {
             // the baseUrl of the resource server
             "window.resourceServerBaseUrl = 'http://localhost:8000';\n";
         }
+
+        @GetMapping("favicon.ico")
+        @ResponseBody
+        void returnNoFavicon() { }
```

Comment:
Can you explain why this is needed?

Reply:
To make browser console clearer and not showing 404 for GET /favicon.ico

Reply:
To make browser console clearer and not showing 404 for GET /favicon.ico